front/app/containers/Admin/reporting/components/ReportBuilder/Analysis/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useCallback, useState } from 'react';
+
+import { Box } from '@citizenlab/cl2-component-library';
+import { IOption } from 'typings';
+
+import usePhase from 'api/phases/usePhase';
+
+import { useReportContext } from 'containers/Admin/reporting/context/ReportContext';
+
+import PhaseFilter from 'components/UI/PhaseFilter';
+
+import { useIntl } from 'utils/cl-intl';
+
+import ProjectFilter from '../Widgets/_shared/ProjectFilter';
+import QuestionSelect from '../Widgets/_shared/QuestionSelect';
+import widgetMessages from '../Widgets/messages';
+
+import Analyses from './Analyses';
+import messages from './messages';
+
+const Analysis = ({ selectedLocale }: { selectedLocale: string }) => {
+  const { projectId: contextProjectId, phaseId: contextPhaseId } =
+    useReportContext();
+
+  const [projectId, setProjectId] = useState<string | undefined>(
+    contextProjectId
+  );
+  const [phaseId, setPhaseId] = useState<string | undefined>(contextPhaseId);
+  const [questionId, setQuestionId] = useState<string | undefined>(undefined);
+
+  const { data: phase } = usePhase(phaseId);
+
+  const { formatMessage } = useIntl();
+
+  const handleProjectFilter = useCallback(({ value }: IOption) => {
+    setProjectId(value);
+    setPhaseId(undefined);
+    setQuestionId(undefined);
+  }, []);
+
+  const handlePhaseFilter = useCallback(({ value }: IOption) => {
+    setPhaseId(value);
+    setQuestionId(undefined);
+  }, []);
+
+  const handleQuestion = useCallback((questionId: string) => {
+    setQuestionId(questionId);
+  }, []);
+
+  const isNativeSurveyPhase =
+    phase?.data.attributes.participation_method === 'native_survey';
+
+  const showAnalyses = isNativeSurveyPhase
+    ? projectId && phaseId && questionId
+    : projectId;
+
+  return (
+    <Box>
+      <ProjectFilter
+        projectId={projectId}
+        emptyOptionMessage={widgetMessages.noProject}
+        onProjectFilter={handleProjectFilter}
+      />
+
+      {projectId !== undefined && (
```

Comment:
The way you select what kind of insights you want to see (for ideation you select the whole project, but then for a survey you have to select a phase) seems a bit counterintuitive. I understand why it works like that because I understand the technical background, but as a user it's a bit strange that for ideation I just have to select the project, but for a survey I have to select a phase and even a specific question